src/main/java/ev3dev/sensors/ev3/EV3ColorSensor.java

```diff
@@ -65,10 +63,11 @@ private void initModes() {
         				new AmbientMode(this.PATH_DEVICE)
         		});

```

Comment:
Hmmmm, this way of doing things will screw up logic in the BaseSensor class. We have to first register the modes through `setModes()` and then get them through `getMode()`.